test/io/db_config.sql

```diff
@@ -1,75 +1,77 @@
 CREATE ROLE db_config_authenticator LOGIN NOINHERIT;
 
 -- reloadable config options
-ALTER ROLE db_config_authenticator SET pgrst.jwt_aud = 'https://example.org';
-ALTER ROLE db_config_authenticator SET pgrst.openapi_server_proxy_uri = 'https://example.org/api';
-ALTER ROLE db_config_authenticator SET pgrst.jwt_secret = 'REALLY=REALLY=REALLY=REALLY=VERY=SAFE';
-ALTER ROLE db_config_authenticator SET pgrst.jwt_secret_is_base64 = 'false';
-ALTER ROLE db_config_authenticator SET pgrst.jwt_role_claim_key = '."a"."role"';
-ALTER ROLE db_config_authenticator SET pgrst.jwt_cache_max_lifetime = '3600';
+-- these settings will override the values in configs/no-defaults.config, so they must be different
```

Comment:
I noticed that some values used here were the same than the ones in the `no-defaults.config`. Since we want to test if they are overridden correctly, then the values must be different from each other, that's why I changed them (also reordered the settings alphabetically while I'm at it).